ReactiveXComponent/Connection/IXCPublisher.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using ReactiveXComponent.Common;
+
+namespace ReactiveXComponent.Connection
+{
+    public interface IXCPublisher : IDisposable
+    {
+        void InitPrivateCommunication(string privateCommunicationIdentifier);
```

Comment:
not sure it's the right place to init private communication.
I think we want to init it once, no ?